server.js

```diff
@@ -5,10 +5,40 @@
 
 // Today is <day name>, <month name> <date>. Here are the available food trucks:
 
-// list the food trucks returned by filterByDay, passing in the current day name.
-// e.g. filterByDay(day); where "day" is determined using the enhanced-date module
-// The list of trucks returned will be an array of food truck objects. Iterate 
-// through the list, building up a string of food truck names. Once you're done 
-// iterating through that list, display the string you built up.
+//list the food trucks returned by filterByDay, passing in the current day name.
+// e.g. filterByDay(day); where "day" is determined using the enhanced-date
+// module. The list of trucks returned will be an array of food truck objects.
+// Iterate through the list, building up a string of food truck names. Once
+// you're done iterating through that list, display the string you built up.
 
-// Remember that the response is a stream object that must be closed.
\ No newline at end of file
+// Remember that the response is a stream object that must be closed.
+
+var trucks = require('./trucks.js');
+var enhancedDate = require('./enhanced-date.js');
+var http = require('http');
+enhancedDate.setDate();
```

Comment:
This is better done inside the request handler. The server can run for days, but the date will always be set to when the server started by doing this here.